src/main/resources/config/application-dev.yml

```diff
@@ -40,34 +45,40 @@ spring:
             kafka:
                 binder:
                     brokers: localhost
+                    zk-nodes: localhost
                     defaultBrokerPort: 9092
+            bindings:
+                output:
+                    destination: topic-jhipster
     datasource:
         type: com.zaxxer.hikari.HikariDataSource
-        url: jdbc:h2:mem:scheduler;DB_CLOSE_DELAY=-1
-        username: scheduler
-        password:
-    h2:
-        console:
-            enabled: true
+        url: jdbc:postgresql://localhost:5432/scheduler
+        username: postgres
+        password: postgres
     jpa:
-        database-platform: io.github.jhipster.domain.util.FixedH2Dialect
-        database: H2
-        show-sql: true
+        database-platform: io.github.jhipster.domain.util.FixedPostgreSQL82Dialect
```

Comment:
why do you change H2 DB for test to Postgres?

Reply:
fixed